src-relution/mwFileUpload/mwFileUpload.js

```diff
@@ -267,6 +275,33 @@ angular.module('mwFileUpload', [])
             }.bind(this));
           },
           add: function (e, data) {
+            var hasFileSizeError = false;
+            if (scope.maxFileSizeByte !== undefined) {
```

Comment:
if (scope.maxFileSizeByte) {

Reply:
ah yeah, I thought about someone setting it to `0` - but that makes not too much sense as a use case.